include/envoy/access_log/access_log.h

```diff
@@ -62,6 +62,7 @@ class Instance {
    *                      the request headers.
    */
   virtual void log(const Http::HeaderMap* request_headers, const Http::HeaderMap* response_headers,
+                   const Http::HeaderMap* response_trailers,
```

Comment:
please update method docs, same below/elsewhere

Reply:
done. Seems like it was the only place though:)